pkg/executor/runner/runner.go

```diff
@@ -299,23 +303,25 @@ func (r *Runner) WriteConfigFile() error {
 		return err
 	}
 
-	err = writer.Write(jsonConfig, "/root/module/config.tf.json")
+	err = writer.Write(jsonConfig, fmt.Sprintf("%s/config.tf.json", path))
```

Comment:
Please use `path.Join()` here instead of concatenation.

Reply:
Done